listenbrainz/db/user.py

```diff
@@ -515,21 +515,24 @@ def report_user(reporter_id: int, reported_id: int, reason: str = None):
         })
 
 
-def update_user_email(musicbrainz_id, email):
+def update_user_details(lb_id: int, musicbrainz_id: str, email: str):
```

Comment:
does this method have a test? Looks like an easy win that we can add

Reply:
Added.